Bash (AzureDevops).sublime-syntax

```diff
@@ -16,3 +16,8 @@ contexts:
   string-prototype:
     - meta_prepend: true
     - include: scope:source.yaml.pipeline.azure-devops#string-interpolations
+
+variables:
```

Comment:
I think the general/common order of main keys inside syntax definition should be followed here, i.e. define `variables` before `contexts`. Same for the other syntax.

Reply:
Maybe not primarily relevant here, but

Q: https://github.com/sublimehq/Packages/pull/3896#issuecomment-1944709585

A: https://github.com/sublimehq/Packages/pull/3896#issuecomment-1944709585

Already moved variables to bottom in Lisp. Same done in Bash/Zsh.